assets/css/admin.scss

```diff
@@ -1,8 +1,13 @@
+$govuk-compatibility-govuktemplate: true;
+$govuk-images-path: "../build/node_modules/govuk-frontend/govuk/assets/images/";
+$govuk-fonts-path: "../build/node_modules/govuk-frontend/govuk/assets/fonts/";
+@import "govuk-frontend/govuk/all";
+
 .hallo-editor {
   @import "common/variables";
   @import "common/mixins";
   @import "common/common";
-  @extend .entry-content;
+  @extend .entry-content !optional;
```

Comment:
TIL about [this feature](https://sass-lang.com/documentation/at-rules/extend#mandatory-and-optional-extends). Thanks!

But can I ask why we're using it here? Does this need to be optional for the sake of the child theme?

Reply:
Removed again optional, it was removed in a previous branch but not this one.